packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -494,9 +495,13 @@ class _FlutterPlatform extends PlatformPlugin {
       printTrace('test $ourTestCount: awaiting initial result for pid ${process.pid}');
       final _InitialResult initialResult = await Future.any(<Future<_InitialResult>>[
         process.exitCode.then<_InitialResult>((int exitCode) => _InitialResult.crashed),
-        timeout.future.then<_InitialResult>((Null _) => _InitialResult.timedOut),
+        timeout.future.then<_InitialResult>((void value) => _InitialResult.timedOut),
```

Comment:
Here and below: any reason not to use `_` for unused params? If so, we should probably document in the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) wiki.

Reply:
`_` doesn't say what the argument is (nor does `value` particularly in this case, but at least it's clear that it's _not_ something like "elapsedTime" or "error").

The style guides (Flutter and Dart) don't seem to suggest using `_`. (I think Flutter's used to, but that changed a while ago.)

Reply:
Interesting - you're right; I'd thought Effective Dart mentioned this at one point, but after a quick look, it doesn't. It's a bit of an anti-pattern for method overrides and function defs that are intended to match a typedef in any case, which limits it to lambdas with very obvious params. SGTM; thanks for clarifying.